.github/workflows/build.yml

```diff
@@ -29,13 +29,13 @@ jobs:
     - name: Get Short Sha
       id: short-sha
       run: echo "::set-output name=sha::$(git rev-parse --short=12 HEAD)"
-    - id: 'auth'
-        name: 'Authenticate to Google Cloud'
-        uses: 'google-github-actions/auth@v1'
-        with:
-          credentials_json: '${{ secrets.GCR_PUBLISH_KEY }}'
-      - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@v0
+    - name: 'Authenticate to Google Cloud'
+      id: 'auth'
```

Comment:
does this field (as well as `name` and `uses` and `credentials_json`, I suppose) need these single quotes? Other entries appear to omit them; I wonder if the id of this step ends up being `auth` or `'auth'` and if there's a difference.

Reply:
This is probably a nit, since this was how the values were before this PR.

Reply:
I think for consistencies sake I should match the rest of the file. I'll make the changes.